docs/configuring-output.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[output-config]]
+== Output Configuration
+
+An output direct data from APM Server to the desired storage system.
+A typical deployment uses Elasticsearch as the output.
+Options like Logstash and Kafka are popular with users requiring additional data enrichment and highly scalable setups.
+
+[float]
+=== Sourcemaps
+
+Sourcemaps can be uploaded through various outputs but can currently only be applied when Elasticsearch is the output.
+
+[[libbeat-configuration-fields]]
+[float]
+== `fields`
+
+Fields are optional tags that can be added to any output.
+Fields can be scalar values, arrays, dictionaries, or any nested combination of these.
+By default, the fields that you specify here will be grouped under a `fields` sub-dictionary in the output document.
+
+Example:
+
+[source,yaml]
+------------------------------------------------------------------------------
+fields: {project: "myproject", instance-id: "574734885120952459"}
+------------------------------------------------------------------------------
+
+To store the custom fields as top-level fields, set the `fields_under_root` option to true.
+This is not recommended as backwards compatibility can not be ensured as new fields are added to APM documents.
+
+include::./copied-from-beats/outputconfig.asciidoc[]
```

Comment:
Since this included file is on the same level as `output-config` it creates a separate html page, leading to 

<img width="729" alt="screen shot 2018-07-10 at 10 46 57" src="https://user-images.githubusercontent.com/5555349/42499263-9eb93224-842e-11e8-9477-44754c63e94a.png">

I suggest to add a check in the libbeat file if there is an additional config file like e.g. `beats-specific-output-config` and include it if defined. I quickly tried it out and it would give you something like this:

<img width="650" alt="screen shot 2018-07-10 at 11 02 36" src="https://user-images.githubusercontent.com/5555349/42500180-d99330a0-8430-11e8-8e24-b9c3c92a4728.png">

by adding:

* index.asciidoc:
`:beats-specific-output-config: ./../configuring-output.asciidoc`
* docs/copied-from-beats/outputconfig.asciidoc:
```
ifdef::beats-specific-output-config[]
  include::{beats-specific-output-config}[]
endif::[]
```
* docs/configuring.asciidoc
```
-include::./configuring-output.asciidoc[]
+include::./copied-from-beats/outputconfig.asciidoc[]
+include::./copied-from-beats/shared-config-ingest.asciidoc[]
```
* docs/configuring-output.asciidoc
```
-== Output Configuration

-== `fields`
+=== `fields`

-include::./copied-from-beats/outputconfig.asciidoc[]
-include::./copied-from-beats/shared-config-ingest.asciidoc[]
```

Reply:
thanks, this is a great idea